bin/setup_dev

```diff
@@ -58,6 +58,11 @@ $AG install \
     python$PVERSION python3-pkg-resources python3-setuptools \
     python$PVERSION-dev python3-pip python emacs-nox python$PVERSION-venv nmap
 
+PYTHON_BIN=$(readlink -f $(which python))
+echo Python binary is at $PYTHON_BIN
+sudo ln -fs $PYTHON_BIN /usr/bin/python
```

Comment:
this is dangerous and really not a good idea -- you're assuming that this is the only thing that will ever run on the target system, which is not a good idea... and could seriously mess up the platform otherwise.

What scope is this meant to cover?  If it's for just testing then it could be something specific in the github actions page.  

If it's really something with mininet, then we might be able to push/make a change to that package to do the needful (e.g. call python3 directly), or allow somehow running in a venv or with a specific python version.

Another alternative would be to put something in the binhack/ directory and add that to the PATH environment... but that wouldn't work if the mininet executable is being run sudo (which resets the path).